flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/validate/FlinkSqlConformance.java

```diff
@@ -166,4 +166,14 @@ public boolean allowQualifyingCommonColumn() {
     public SqlLibrary semantics() {
         return SqlConformanceEnum.DEFAULT.semantics();
     }
+
+    @Override
+    public boolean allowCoercionStringToArray() {
+        return SqlConformanceEnum.DEFAULT.allowCoercionStringToArray();
+    }
+
+    @Override
+    public boolean isValueAllowed() {
+        return SqlConformanceEnum.DEFAULT.isValueAllowed();
```

Comment:
check if false, but it would be fine if it is true

Reply:
based on tests we could enable it